dom.bs

```diff
@@ -4271,7 +4287,10 @@ invoked, must run these steps:
  "{{NotSupportedError!!exception}}" {{DOMException}}.
 
  <li><p>Return a <a lt="clone a node">clone</a> of <a>this</a>, with the
- <i>clone children flag</i> set if <var>deep</var> is true.
+ <i>clone children flag</i> set if <var>deep</var> is true, and the
+ <i>clone shadows flag</i> set if <a>this</a> is a {{DocumentFragment}} whose
```

Comment:
It's really bizarre to special case when calling clone on template element's content fragment.

Reply:
Yeah, I don't disagree that this is a bit odd. But it's necessary (or something similar) to allow a "normal" `<template>` element to contain a declarative shadow root and get cloned properly into DOM. If you have suggestions for how to better implement that capability, I'm open to them!